WeatherForcast/WeatherForcast/Scenes/HomeScreen/HomeViewController.swift

```diff
@@ -123,8 +223,13 @@ extension HomeViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
 
         let delete = UIContextualAction(style: .destructive, title: "Xoá") { [weak self] (action, view, nil) in
-            self?.weatherList.remove(at: indexPath.row)
-            self?.homeTableView.reloadData()
+
+            self?.deleteObjectRealm(with: self?.weatherList[indexPath.row], onSuccess: {
+                self?.weatherList.remove(at: indexPath.row)
+                self?.homeTableView.reloadData()
+            }, onFailed: { [weak self] error in
+                self?.alertShow(title: Message.errorTitle, message: "\(error)", view: self ?? UIViewController())
```

Comment:
```suggestion
                self?.alertShow(title: Message.errorTitle, message: "\(error.localizedDescription)", view: self ?? UIViewController())
```
tương tự với những cái khác